### PR TITLE
Handle create accout with google

### PR DESCRIPTION
**Pull request by duccnv-1684**
https://edu-redmine.framgia.vn/issues/4099

### CODE REVIEW

**Comment by duccnv-1684** [comment]
ready

File: app/src/main/java/com/framgia/f_talk/screen/signup/SignUpActivity.java [hunk 1]
```diff
@@ -80,6 +127,63 @@ public void moveToCreateAccountActivity() {
...
+        super.onActivityResult(requestCode, resultCode, data);
+        if (requestCode == SIGN_UP_WITH_GOOGLE_CODE) {
+            Task<GoogleSignInAccount> task = GoogleSignIn.getSignedInAccountFromIntent(data);
+            try {
+                GoogleSignInAccount account = task.getResult(ApiException.class);
+                AuthCredential credential = GoogleAuthProvider
+                        .getCredential(account.getIdToken(), null);
+                mSignUpViewModel.signUpWithGoogle(FirebaseAuth.getInstance(), credential);
+            } catch (ApiException e) {
+                e.printStackTrace();
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/f_talk/screen/signup/SignUpActivity.java]: handle Ex

File: app/src/main/java/com/framgia/f_talk/screen/signup/SignUpActivity.java [hunk 2]
```diff
@@ -80,6 +127,64 @@ public void moveToCreateAccountActivity() {
...
+    public void onSuccess(LoginResult loginResult) {
+        AuthCredential credential = FacebookAuthProvider
+                .getCredential(loginResult.getAccessToken().getToken());
+        mSignUpViewModel.signUpWithFacebook(FirebaseAuth.getInstance(), credential);
+    }
+
+    @Override
+    public void onCancel() {
+        Toast.makeText(this, getString(R.string.msg_sign_up_cancel), Toast.LENGTH_SHORT).show();
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/f_talk/screen/signup/SignUpActivity.java]: remove  bl

File: app/src/main/java/com/framgia/f_talk/screen/signup/SignUpActivity.java [hunk 3]
```diff
@@ -80,6 +127,64 @@ public void moveToCreateAccountActivity() {
...
+    }
+
+    @Override
+    public void onError(FacebookException error) {
+        Toast.makeText(this, getString(R.string.msg_create_account_fail), Toast.LENGTH_SHORT).show();
+
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/framgia/f_talk/screen/signup/SignUpActivity.java]: remove bl

**Pull request merged**